snippets/terraform-variables.adoc

```diff
@@ -0,0 +1,172 @@
+// Snippets included in the following assemblies and modules:
```

Comment:
I don't know if this renders properly in the preview. For me, it is mostly a dark grey block with a couple of lines of text at the top.